ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/431__data-science-pipelines-api.robot

```diff
@@ -0,0 +1,70 @@
+*** Settings ***
+Documentation       Test suite for OpenShift Pipeline
+Resource            ../../../Resources/RHOSi.resource
+Resource            ../../../Resources/ODS.robot
+Resource            ../../../Resources/Common.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Library             DateTime
+Library             ../../../../libs/DataSciencePipelinesAPI.py
+Suite Setup         Data Science Pipelines Suite Setup
+Suite Teardown      RHOSi Teardown
+
+
+*** Variables ***
+${REDHAT_OPENSHIFT_PIPELINES_YAML}=            tests/Resources/Files/redhat-openshift-pipelines.yaml
+${DATA_SCIENCE_PIPELINES_APPLICATION_YAML}=    tests/Resources/Files/data-science-pipelines-sample.yaml
```

Comment:
These need to be updated to `ods_ci/tests/Resources/...` in order to work when running ods_ci as `./ods_ci/run_robot_tests.sh`